api/src/main/resources/liquibase-schema-only.xml

```diff
@@ -2429,6 +2429,12 @@
     <changeSet author="ben (generated)" id="1227303685425-75">
         <addPrimaryKey columnNames="concept_id, concept_set" tableName="concept_set_derived"/>
     </changeSet>
+    <changeSet author="vshankar" id="201609171146-1">
```

Comment:
I figured it out, I had to drop the foreign key constraint before dropping the primary key. Thanks.